uint8 STOP = 0
uint8 FORWARD = 1
uint8 BACKWARD = 2
uint8 LEFT = 3
uint8 RIGHT = 4

# One of the four commands above
uint8 cmd
int32 speed_qpps  # Speed in Quadrature pulses per second
